pkg/extensions/tbtc/liquidation_recovery_test.go

```diff
@@ -0,0 +1,158 @@
+package tbtc
+
+import (
+	"bytes"
+	cecdsa "crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/hex"
+	"math/big"
+	"testing"
+
+	"github.com/btcsuite/btcd/chaincfg"
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/keep-network/keep-ecdsa/pkg/ecdsa"
+)
+
+func Test_PublicKeyToP2WPKHScriptCode_Works(t *testing.T) {
```

Comment:
Maybe we could also test the result of the conversion with some fixed input and outputs, e.g. based on the ones from [BIP-143](https://github.com/bitcoin/bips/blob/master/bip-0143.mediawiki)?